### PR TITLE
Setup and Motor Class

### PR DESCRIPTION
**Pull request by andrewtran10**
Created setup.py and changed MotorControls.py for robust and efficiency
Removed original motor control methods and created class so we don't have to import back and forth.

### CODE REVIEW

File: Challenge1 Code/main.py [hunk 1]
```diff
@@ -11,32 +11,25 @@
...
 if __name__ == "__main__":
+	# setting up pins
+	setup.setupPins()
+
+	# instantiate motor class
+	Motors = mc.Motors([Constants.IN1, Constants.IN2, Constants.IN3, Constants.IN4])
+
+	Motors.forwards(78)
+
+	Motors.shutdown()
```
Comment (sdub18) [inline Challenge1 Code/main.py]: Why do you call the Motors.shutdown here? Wouldn't we want to reserve that command for the end when we are cleaning up?

File: Challenge1 Code/main.py [hunk 2]
```diff
@@ -47,7 +40,8 @@ def setupPins():
 		# MARK: Setup and Receive Data -----------------
 
 		# Setup Pins
-		setupPins()
+		setup.setupPins()
+		Motors = mc.Motors([Constants.IN1, Constants.IN2, Constants.IN3, Constants.IN4])
 
 		# Set the debug LED to ensure code is getting to robot
```
Comment (sdub18) [inline Challenge1 Code/main.py]: Do we need all of this commented code? Seems like we can get rid of it

**Review by sdub18 — CHANGES_REQUESTED** [review]
This looks like a great addition to the codebase. Makes everything a lot more robust and clear about what is going on. Just have a few things for you to look at before we officially merge this in with master

**Comment by sdub18** [comment]
Yeah It looks like you remove a lot of the logic from the previous commit. I would try and add that code back in with your changes before we push to master

File: Challenge1 Code/main.py (hunk 1 above)
Comment (andrewtran10) [reply Challenge1 Code/main.py]: Just testing the class methods. We can get rid of it in master

Comment (sdub18) [reply Challenge1 Code/main.py]: Ahhhh I see. in that case makes sense

**Pull request merged**